engine/src/main/scala/cromwell/server/CromwellRootActor.scala

```diff
@@ -251,4 +258,8 @@ object CromwellRootActor extends GracefulStopSupport {
   val DefaultNumberOfWorkflowLogCopyWorkers = 10
   val DefaultCacheTTL = 20 minutes
   val DefaultNumberOfCacheReadWorkers = 25
+
+  def callCachingBoolean(path: String, callCachingConfig: Config): Boolean = {
+   callCachingConfig.getBoolean(path)
+  }
```

Comment:
This function only seems to be used in one place, and should probably just be inlined.